### PR TITLE
getTimeSig to getInfo, TimeSignature.timeSig to TimeSignature.info

### PR DESCRIPTION
**Pull request by rvilarl**
As suggested.

### CODE REVIEW

File: src/timesignote.ts [hunk 1]
```diff
@@ -13,8 +13,8 @@ export class TimeSigNote extends Note {
     this.setAttribute('type', 'TimeSigNote');
 
     const timeSignature = new TimeSignature(timeSpec, customPadding);
-    this.timeSig = timeSignature.getTimeSig();
-    this.setWidth(this.timeSig?.glyph.getMetrics().width ?? 0);
+    this.timeSig = timeSignature.getInfo();
```
Comment (0xfe) [inline src/timesignote.ts:16]: Can you rename this to `this.timeSigInfo` so it's clear?

**Comment by 0xfe** [comment]
Looks good, minor comment.

File: src/timesignote.ts (hunk 1 above)
Comment (rvilarl) [reply src/timesignote.ts]: Done

**Comment by 0xfe** [comment]
Looks great, thanks!

**Pull request merged**